backend/apps/rbac/views.py

```
@@ -14,6 +14,64 @@
+    org_ids = set()
+
+    profile = getattr(user, "user_profile", None) or getattr(user, "profile", None)
+    if profile and getattr(profile, "organization_id", None):
+        org_ids.add(profile.organization_id)
+
+    try:
+        user_org = getattr(user, "organization", None)
+        if user_org and getattr(user_org, "id", None):
+            org_ids.add(user_org.id)
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)